include/mxnet/imperative.h

```diff
@@ -191,6 +191,7 @@ class Imperative {
   std::atomic<uint64_t> variable_count_{0};
   /*! \brief default backward bulk size */
   int backward_bulk_size_{0};
+  bool backward_graph_dump_enabled_{};
```

Comment:
not needed.

Reply:
replied above. I think is more efficient and better debuggability to have a flag than to repeatedly pull the environment.